microf/__main__.py

```diff
@@ -236,7 +242,8 @@ def accept(self, filename):
             r'(?P<well_letter>[A-Z]+) - (?P<well_nr>[0-9]+)'
             r'\('
             r'fld (?P<site>[0-9]+)'
-            r' wv (?P<channel_color>[A-Za-z]+) - (?P<channel_tag>.+)'
+            r' wv (?P<channel_color>[A-Za-z]+) - (?P<channel_tag>\w*)'
+            r'( z (?P<z_pos>\d*))*'
```

Comment:
I guess there should be _at most one_ Z-slice specifier, right?  Also, `z` should be followed by _at least one_ digit for the specifier to be correct.

So it would be more correct to use `r'( z (?P<z_pos>\d+))?'` here.

Reply:
Thx, this improves the regex. I added that